internal/command/user_grant.go

```diff
@@ -291,21 +291,14 @@ func (c *Commands) checkUserGrantPreCondition(ctx context.Context, usergrant *do
 	ctx, span := tracing.NewSpan(ctx)
 	defer func() { span.EndWithError(err) }()
 
-	preConditions := NewUserGrantPreConditionReadModel(usergrant.UserID, usergrant.ProjectID, usergrant.ProjectGrantID, resourceOwner)
```

Comment:
I would add a feature flag for user grants

Reply:
I've set it behind the ImprovedPerformanceTypeProjectGrant flag, as it uses exactly the same search